CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## v0.6.1 (tbd)
   - allow copying from Depth32Float textures
+  - fix for mixed STORAGE_STORE and STORAGE_LOAD
```

Comment:
nit: since `STORAGE_STORE` is not something users see, we should rather say something like "fix usage tracking of read-only buffer storages"

Reply:
Ah, I see. I just copied the commit message. I'll fix that.